neurodsp/plts/time_series.py

```diff
@@ -47,18 +47,24 @@ def plot_time_series(times, sigs, labels=None, colors=None, ax=None, **kwargs):
 
     ax = check_ax(ax, (15, 3))
 
-    times = repeat(times) if isinstance(times, np.ndarray) else times
-    sigs = [sigs] if isinstance(sigs, np.ndarray) else sigs
+    n_repeats = len(sigs) if isinstance(sigs, list) or sigs.ndim == 2 else 1
```

Comment:
what is the goal / benefit of making an explicit number of repeats, for colors / labels / times. 
The idea before was that the limiting factor for the plot loop should be `sigs` - it will loop for the number of signals available. The benefit of _not_ making an explicit number of repeats, is that if I want to plot every second signal in red or something, I could pass 4 signals with `colors=['black', 'red']`, which would no longer work. Because of this, I'm not sure I see any reason for the explicit repeats, and would probably say step that back?